### PR TITLE
Start React app

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,193 @@
+> *This project was bootstrapped with [Create React App](https://github.com/facebookincubator/create-react-app) +
+  [backpack-react-scripts](https://github.com/Skyscanner/backpack-react-scripts/tree/master/packages/react-scripts).*
+  It is very similar to one that you would create using [Create React App](https://github.com/facebookincubator/create-react-app)
+  without [backpack-react-scripts](https://github.com/Skyscanner/backpack-react-scripts/tree/master/packages/react-scripts),
+  so please refer to [it's documentation](https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md)
+  for more usage information.
+
+# [_Put your project name here_]
```
Comment (Spedge) [inline README.md]: Can you update the name and description in here? Also - can you put something in for 'npm install' as I had to run that before npm start would work.

**Comment by Spedge** [comment]
Do we also want to add a Travis CI build / test step?

File: src/api.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+// const API_END_POINT = ""; // todo: Actual end API end point
+const API_END_POINT = "http://0.0.0.0:8080/"; // For running locally, please see note below:
+/*
+	You will have to disable the same origin policy on Chrome for the API to work with the UI.
+	Note - Close all Chrome windows before running this:
+	$ open -a Google\ Chrome --args --disable-web-security --user-data-dir
+	And to re-enable, close all Chrome windows and restart. It should go back to normal
+*/
+
+function makeRequest(url) {
```
Comment (Spedge) [inline src/api.js:10]: If the server isn't running, Ithe code returns a nasty error. The code should handle that scenario

File: README.md (hunk 1 above)
Comment (ghost) [reply README.md]: Updated readme

File: src/api.js (hunk 2 above)
Comment (ghost) [reply src/api.js:10]: screen shot 2017-12-11 at 8 31 24 pm

**Comment by ghost** [comment]
I'm not too sure how to do Travis CI build / test step

File: src/api.js (hunk 2 above)
Comment (Spedge) [reply src/api.js:10]: Sweeet!

**Review by Spedge — APPROVED** [review]
Happy to take an initial commit and work from it.

**Pull request merged**